making_decisions/numbers_to_names.rb

```diff
@@ -0,0 +1,46 @@
+def info_request(question)
+  puts question
+  gets.chomp
+end
+
+def switch(number)
```

Comment:
there are many ways to write this, f.e.:
```
MONTHS = %w[none Jan Feb March April ...].freeze
def switch(number)
  MONTHS[number]
end
```

or:
```
MONTHS = {1 => 'Jan', 2 => 'Feb', ...}.freeze
def switch(number)
  MONTHS[number]
end
 ```

or:
```
def switch(number)
  Date::ABBR_MONTHNAMES[number]
end
```
(as can be seen here: https://ruby-doc.org/stdlib-2.3.1/libdoc/date/rdoc/Date.html)

or:
```
def switch(number)
  Date::MONTHNAMES[number]
end
```

however, when using switch - it will use that many lines. I would suggest however solution with an array.

Reply:
I really like the first solution 👌